test/unit/helpers/xhr/MFA_CHALLENGE_customFactor.js

```diff
@@ -0,0 +1,52 @@
+define({
+  "status": 200,
+  "responseType": "json",
+  "response": {
+    "stateToken": "00M2MpKONl8k_a3oRjwDcznNy6NKeNfb0lkKLUjDVG",
```

Comment:
nit: should be a string with `testStateToken` or something similar, no need for this token here

Reply:
👍 sorry, forgot to change